velox/docs/functions/presto/conversion.rst

```diff
@@ -481,12 +483,39 @@ Valid examples
   SELECT cast(nan() as varchar); -- 'NaN'
   SELECT cast(infinity() as varchar); -- 'Infinity'
   SELECT cast(true as varchar); -- 'true'
-  SELECT cast(timestamp '1970-01-01 00:00:00' as varchar); -- '1970-01-01T00:00:00.000'
+  SELECT cast(timestamp '1970-01-01 00:00:00' as varchar); -- '1970-01-01 00:00:00.000'
   SELECT cast(cast(22.51 as DECIMAL(5, 3)) as varchar); -- '22.510'
   SELECT cast(cast(-22.51 as DECIMAL(4, 2)) as varchar); -- '-22.51'
   SELECT cast(cast(0.123 as DECIMAL(3, 3)) as varchar); -- '0.123'
   SELECT cast(cast(1 as DECIMAL(6, 2)) as varchar); -- '1.00'
 
+From timestamp
+^^^^^^^^^^^^
+
+By default, casting a timestamp to a string returns ISO 8601 format with an empty space
+as separator between date and time.
+
+By default legacy_cast is false. If legacy_cast is set to true, the string will use the character 'T' as separator between
```

Comment:
Drop "By default legacy_cast is false."

If legacy_cast configuration property is true, the result string uses character 'T' as separator between date and time and year part is not padded.

Let's also add legacy_cast to the list of configs in [config.rst](https://facebookincubator.github.io/velox/configs.html) 

> The support for legacy_cast = true will be deprecated in the near future.

Let's remove this sentence as we don't know when this config will be removed.

Reply:
Added it to configs.rst. Thanks!